python/ingestion/merge_utils.py

```diff
@@ -352,18 +352,28 @@ def merge_intersectional_pop(
     - tuple containing the merged DataFrame and the string name of the added intersectional population column.
     """
 
+    if primary_demo_col == std_col.RACE_COL:
+        primary_demo_col = 'race_and_ethnicity'
+
     pop_dtype: dict[str, Union[Type[float], Type[str]]] = {
         std_col.POPULATION_COL: float,
     }
 
+    geo_file = ''
+
     if geo_level == COUNTY_LEVEL:
         pop_dtype[std_col.COUNTY_FIPS_COL] = str
-    if geo_level == STATE_LEVEL:
+        geo_file = 'county'
+    if geo_level in [STATE_LEVEL, NATIONAL_LEVEL]:
         pop_dtype[std_col.STATE_FIPS_COL] = str
+        geo_file = 'state'
```

Comment:
Could we use the predefined constants for this line?

`geo_file = STATE_LEVEL`